### DISCUSSION TITLE
FInicky setup

### DISCUSSION THREAD

**Discussion by hmltn-0** [coddingtonbear/obsidian-web#182] [Bug Report]
### Describe the bug

Awesome extension but i have set it up on two computers and it's quite finicky. I don't know why it sometimes requires quite a bit of fiddling to get it to recognize that the api key is correct. Sometimes I have to first enter an ip address into my browser manually, as 127.0.0.1/xxxxx/active to get the browser to recognize it is already authenticated.

### To Reproduce

Start with a clean installation of Chrome and try installing it.

### Operating System

Ubuntu latest

### Browser

Chrome

### Browser Version

latest

**Comment by coddingtonbear** [comment]
I’m afraid i’m on a very similar setup (chrome on ubuntu) and don’t have any such problem. It’s hard for me to know what you mean by “entering [it] manually”, and how you are testing that the key is accepted, though, given that there are several ways to interpret either of those things. Maybe if you upload a bug report export from the options page I might be able to give some troubleshooting steps to you?
